tests/ContentTransformer/HtmlStylesRemoverTest.php

```diff
@@ -0,0 +1,276 @@
+<?php
+
+namespace EMS\CoreBundle\Tests\ContentTransformer;
+
+use Symfony\Bundle\FrameworkBundle\Test\WebTestCase;
+use EMS\CoreBundle\ContentTransformer\HtmlStylesRemover;
+
+class HtmlStylesRemoverTest extends WebTestCase
+{
+    public function testRemoveSimpleInline()
+    {
+        $input = <<<HTML
+<p>Lorem ipsum dolor <span class="removable-style-newWord">sit amet</span>, consectetur adipiscing elit.</p>
+HTML;
+        $output = <<<HTML
+<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p>
+HTML;
+
+        $htmlStylesRemover = new HtmlStylesRemover();
```

Comment:
Isn't there a "setup" function that can be used for this? (because every test case is using this line of code).